develop/devguide/Connector/ImpactOfProtocolChanges/ImpactOfProtocolVersionChanges.md

```diff
@@ -0,0 +1,103 @@
+- [Change Unicode](xref:ChangeUnicode)
+- [Change connection(s)](xref:ChangeConnections)
+- [Change layout](xref:ChangeLayout)
+- [Change exported protocol name (DVE)](xref:ChangeExportedProtocolName)
+- [Change page name](xref:ChangePageName)
+- [Change minimum required DataMiner version](xref:ChangeMinimumRequiredDMAVersion)
+- [Change protocol name](xref:ChangeProtocolName)
+
+### Parameter changes
+
+- [Change parameter description](xref:ChangeParameterDescription)
+- [Change parameter ID](xref:ChangeParameterID)
+- [Change parameter discreet and/or exception](xref:ChangeParameterDiscreetException)
+- [Change parameter Interprete](xref:ChangeParameterInterprete)
+- [Change parameter range](xref:ChangeParameterRange)
+- [Change matrix parameter size](xref:ChangeMatrixParameterSize)
+- [Change alarm monitoring and/or trending](xref:ChangeAlarmMonitoringTrending)
+- [Remove parameter](xref:RemoveParameter)
+
+### Table parameter changes
+
+- [Change primary Key](xref:ChangePrimaryKey)
+- [Change display key](xref:ChangeDisplayKey)
+- [Replace displayColumn by naming](xref:ReplaceDisplayColumnByNaming)
+- [Change to partial table](xref:ChangeToPartialTable)
+- [Change logger table](xref:ChangeLoggerTable)
+- [Change column order](xref:ChangeColumnOrder)
+- [Change displayed column order](xref:ChangeDisplayedColumnOrder)
+
+## Improving context awareness procedure
+
+A procedure has been introduced to improve context awareness and to allow certain major changes to improve the overall quality of protocols even though they could cause impact.
+
+### Context awareness
+
+Any user familiar with a product should easily find all necessary information when using DataMiner protocols. This means that DataMiner protocols should not only perform great, but should also look the part and be straightforward to use.
+
+Especially for new developments, the focus should be on getting them spot on from the start, especially since any changes that might be needed in the future could cause impact for existing users. 
+
+However, this should not stop you from improving existing protocols.
+
+### Major changes
+
+In the past, as many major changes as possible were blocked by providing workarounds or convincing the users to not make certain changes to avoid impact. While this worked, this meant new users might not get a perfect product. To improve the quality of DataMiner protocols, a different approach is therefore needed.
+
+### Procedure
+
+Whenever you receive feedback from a user that some improvements (changing parameter descriptions, change layout, etc.) could be done in a protocol, a major change can be requested to allow these changes.
+
+As a Skyline developer, you should create such requests on the [Protocol Version Request](https://intranet.skyline.be/Lists/Protocol%20Version%20Request/Overview.aspx) intranet page (similar to any other protocol version request), but no specific task is needed (unless you already have a task in your project).
```

Comment:
Is this information still accurate now that intranet is getting retired?

Reply:
for now, this can be replaced with a link to this Teams: https://teams.microsoft.com/l/message/19:ad852d24461147ab918738dc0d3384d8@thread.skype/1695297680965?tenantId=5f175691-8d1c-4932-b7c8-ce990839ac40&groupId=38eed67d-cb0d-47f7-a6ea-d7487cc35ce3&parentMessageId=1695297028950&teamName=Expert%20Hub%20-%20Scripts%20%26%20Connectors&channelName=02.%20ECS%20-%20Protocol%20Versions&createdTime=1695297680965

Later on there will be a page for it on internal docs but in the meantime, let's refer to the Teams post which contains all necessary info.

Reply:
Switched it